### DISCUSSION TITLE
Can't add periodic task from Django Admin

### DISCUSSION THREAD

**Discussion by radokristof** [celery/django-celery-beat#555] [Q&A]
### Summary:

Cannot add a periodic task from the admin panel. When I try to select one option from 'Interval', 'Cron', etc... it says that I should select a valid choice even if there is one selected...

* Celery Version: v5.2.3
* Celery-Beat Version: v2.2.1

### Exact steps to reproduce the issue:
1. Create an Interval or Cron schedule.
2. Add a new periodic task.
3. Click Save.

### Detailed information

Here is a screenshot of the error:

![Screenshot (366)](https://user-images.githubusercontent.com/36210568/153513717-4fcbddfa-d2fe-4d13-8453-c5fcd83fabf9.png)

**Comment by radokristof** [comment]
Anyone?

**Comment by auvipy** [comment]
you didn't fill out many related fields

**Comment by radokristof** [comment]
What many fields? I thought that you have to choose one from interval/crontab, etc... This is what makes sense.
Or should I select a schedule for each? What if I only want it to be an interval job.

**Comment by radokristof** [comment]
![image](https://user-images.githubusercontent.com/36210568/172061801-76111969-2bd6-4d23-b52f-293f37a4a6b4.png)

**Comment by auvipy** [comment]
select the read two empty fields. this kind of question is for stackoverflow.

**Reply by radokristof** [reply]
What is 'read two empty field'?
